### PR TITLE
Colorscale drawing fix

### PR DESCRIPTION
**Pull request by etpinard**
fixes https://github.com/plotly/plotly.js/issues/1109

while :hocho: a bunch of useless code :tada:

### CODE REVIEW

File: src/components/drawing/index.js [hunk 1]
```diff
@@ -290,50 +289,22 @@ drawing.pointStyle = function(s, trace) {
...
-        max = maxProp.get();
-
-    // TODO handle this in Colorscale.calc
-    if(scl && Array.isArray(colorArray)) {
-        if(auto || !isNumeric(min) || !isNumeric(max)) {
-            min = Infinity;
-            max = -Infinity;
-            colorArray.forEach(function(color) {
-                if(isNumeric(color)) {
-                    if(min > color) min = +color;
```
Comment (etpinard) [inline src/components/drawing/index.js]: This block here used to override the [`Colorscale.calc`](https://github.com/plotly/plotly.js/blob/master/src/components/colorscale/calc.js) computations in the case where `marker.cauto` (or `marker.line.cauto`) was true. 

:hocho: :hocho: :hocho: 

Luckily, for most cases, this block here gave the same results as `Colorscale.calc`. One exception coming when the computated `cmin` and `cmax` where equal. In that case, `Colorscale.calc` spreads them apart by -0.5 and +0.5 respectively [here](https://github.com/plotly/plotly.js/blob/8745a22e42933b48b85ae05c7daf46bc20e7726c/src/components/colorscale/calc.js#L43-L46) whereas the removed block left them as is.

Comment (alexcjohnson) [inline src/components/drawing/index.js]: sounds right, do you want to test it? Could just be adding a second scale to the mock you already edited...

Comment (etpinard) [inline src/components/drawing/index.js]: good call!

Comment (etpinard) [inline src/components/drawing/index.js]: > do you want to test it?

Test _what_ exactly?

The equal _cmin and cmax_ case is now tested via https://github.com/plotly/plotly.js/pull/1112/commits/860cceec67f6c79057cfea52517233a6b9734953

Comment (etpinard) [inline src/components/drawing/index.js]: Oh ok. You mean show the colorbar. Good call.

Comment (alexcjohnson) [inline src/components/drawing/index.js]: haha yeah, colorscale, colorbar... you figured out what I meant :)

**Comment by alexcjohnson** [comment]
Nice. 💃

**Pull request merged**